keel-web/src/test/kotlin/com/netflix/spinnaker/keel/rest/ApplicationControllerTests.kt

```diff
@@ -171,17 +171,17 @@ internal class ApplicationControllerTests : JUnit5Minutests {
         deliveryConfig.environments.flatMap { it.resources }.forEach { resource ->
           resourceRepository.appendHistory(ResourceValid(resource))
         }
-        artifactRepository.store(artifact, "1.0.0", ArtifactStatus.RELEASE)
-        artifactRepository.store(artifact, "1.0.1", ArtifactStatus.RELEASE)
-        artifactRepository.store(artifact, "1.0.2", ArtifactStatus.RELEASE)
-        artifactRepository.store(artifact, "1.0.3", ArtifactStatus.RELEASE)
-        artifactRepository.markAsSuccessfullyDeployedTo(deliveryConfig, artifact, "1.0.0", "test")
-        artifactRepository.markAsSuccessfullyDeployedTo(deliveryConfig, artifact, "1.0.0", "staging")
-        artifactRepository.markAsSuccessfullyDeployedTo(deliveryConfig, artifact, "1.0.0", "production")
-        artifactRepository.markAsSuccessfullyDeployedTo(deliveryConfig, artifact, "1.0.1", "test")
-        artifactRepository.markAsSuccessfullyDeployedTo(deliveryConfig, artifact, "1.0.1", "staging")
-        artifactRepository.markAsSuccessfullyDeployedTo(deliveryConfig, artifact, "1.0.2", "test")
-        artifactRepository.approveVersionFor(deliveryConfig, artifact, "1.0.3", "test")
+        artifactRepository.store(artifact, "fnord-1.0.0-h0.2add2c9", ArtifactStatus.RELEASE)
+        artifactRepository.store(artifact, "fnord-1.0.1-h1.2add2c9", ArtifactStatus.RELEASE)
+        artifactRepository.store(artifact, "fnord-1.0.2-h2.2add2c9", ArtifactStatus.RELEASE)
+        artifactRepository.store(artifact, "fnord-1.0.3-h3.2add2c9", ArtifactStatus.RELEASE)
```

Comment:
Nit, but supposedly each version would have originated from a different commit?

Reply:
Also, consider adding Docker artifacts to these tests for added coverage.

Reply:
Yes but copy pasting things is hard lol and that didn't add much value.

Also, yes! Adding docker is on my to-do list, which I noted in my PR description :) I'm tracking down a bug that shows up when i have both a deb and a docker docker image, so it's great motivation to add a test for that. I wanted to keep this PR small and focused though